src/games/strategy/triplea/ui/MapData.java

```diff
@@ -31,7 +31,7 @@
 import games.strategy.triplea.ResourceLoader;
 import games.strategy.triplea.image.UnitImageFactory;
 import games.strategy.ui.Util;
-import games.strategy.util.PointFileReaderWriter;
+import tools.map.making.PointFileReaderWriter;
```

Comment:
I'd say for this PR just move it back to core. Can always do a separate PR to split it up.